pkg/cmd/auth/status/status.go

```diff
@@ -141,11 +141,13 @@ func NewCmdStatus(f *cmdutil.Factory, runF func(*StatusOptions) error) *cobra.Co
 		Use:   "status",
 		Args:  cobra.ExactArgs(0),
 		Short: "View all accounts and authentication status",
-		Long: heredoc.Doc(`Verifies and displays information about your authentication state.
+		Long: heredoc.Docf(`
+			Displays information about your authentication state on each known GitHub host.
 
-			This command will test your authentication state for each GitHub host that gh knows about and
-			report on any issues.
-		`),
+			For each host, the authentication state of each known account is tested and any issues are included in
+			the output. Each host section will indicate the active account, which will be used when targeting that host.
+			To change the active account for a host, see %[1]sgh auth switch%[1]s.
```

Comment:
```suggestion

			To change the active account for a host, see %[1]sgh auth switch%[1]s.
```

Reply:
I don't think so, it's directly related to the line above about active accounts.